packages/button/src/css/index.js

```diff
@@ -3,7 +3,7 @@ import {
   defaultName as themeDefaultName,
   names as themeNames
 } from '@pluralsight/ps-design-system-theme/vars'
-import { transparentize } from 'polished'
+import { transparentize } from '@pluralsight/ps-design-system-util/color'
```

Comment:
what type of things do you think should go in `util`? vs a separate package?

Reply:
Things that are util, small, insignificant enough to not warrant the ceremony of a specific package...  That's a rather loose definition.